### PR TITLE
Create ManageServerCard component

### PR DESCRIPTION
**Pull request by mwgiannini**
#147

### CODE REVIEW

**Comment by JoltCode** [comment]
Could we keep this open until the new design system pr is merged? Otherwise we'll have to open a new issue/pr to make this work with the new design system anyway @RhysSullivan

**Comment by RhysSullivan** [comment]
Yeah that makes sense @JoltCode - @mwgiannini nice work on this PR I did a quick skim of the code and it looks pretty solid

**Comment by JoltCode** [comment]
Yep - nice work!

**Comment by JoltCode** [comment]
Depends on #146

File: packages/ui/src/components/ManageServerCard.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      <div className="col-span-2 col-start-1 row-span-2 row-start-1">
+        <ServerIconWithBlurredBackground />
+      </div>
+      <div className="col-span-2 col-start-1 row-span-1 row-start-3">
+        <ServerNameAndRole />
+      </div>
+      <div className="col-span-1 col-start-2 row-span-1 row-start-3 flex">
+        <div className="ml-auto">
+          <Link href={`https://discord.gg/`} target={"Blank"} referrerPolicy="no-referrer">
+            <Button intent={"primary"} visualOnly>
```
Comment (JoltCode) [inline packages/ui/src/components/ManageServerCard.tsx:50]: This now needs to be updated to the new button (path should be the same, just different props and different styles

**Review by JoltCode — CHANGES_REQUESTED** [review]
Really great work (had a proper look), just now needs a little update with the merge of #146 👍

**Comment by JoltCode** [comment]
Closes #147

**Comment by mwgiannini** [comment]
Sounds good, thanks Jolt

**Comment by JoltCode** [comment]
👍

**Comment by mwgiannini** [comment]
@JoltCode @RhysSullivan This should be ready to merge.

File: packages/ui/src/components/ManageServerCard.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    <div className="grid max-w-xs grid-cols-2 grid-rows-3 gap-3 rounded-lg">
+      <div className="col-span-2 col-start-1 row-span-2 row-start-1">
+        <ServerIconWithBlurredBackground />
+      </div>
+      <div className="col-span-2 col-start-1 row-span-1 row-start-3">
+        <ServerNameAndRole />
+      </div>
+      <div className="col-span-1 col-start-2 row-span-1 row-start-3 flex">
+        <div className="ml-auto">
+          <Link href={`https://discord.gg/`} target={"Blank"} referrerPolicy="no-referrer">
```
Comment (JoltCode) [inline packages/ui/src/components/ManageServerCard.tsx:49]: Will open an issue for this - we shouldn't have buttons inside links (needs a component change for button)

**Review by JoltCode — APPROVED** [review]
LGTM

**Pull request merged**